specification/quantumleap.yml

```diff
@@ -3,7 +3,6 @@ info:
   title: "QuantumLeap API"
   version: "0.7.5"      # we'll keep it aligned with QL version
 host: "localhost:8668"  # it'll run in the same container, hence localhost.
-basePath: /v2
```

Comment:
why get rid of the base path? without it, we'll have to be explicit like you've done below, i.e. we'll have to use

    /v2/
    /v2/config
    /v2/version
    ...

instead of 

    /
    /config
    /version
    ...

So that changing version number will require updating all paths...but I'm sure there must a good reason for getting rid of the base path?

Reply:
@c0c0n3 i actually think @iamarnavgarg is right, in the sense, that '/version' '/config' and '/health' should not be under '/v2' because they are not related with NGSI